src/components/App/App.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import HomePermalink from "COMPONENTS/HomePermalink/";
 import HistoryChannel from "COMPONENTS/HistoryChannel/";
 import WatchNow from "COMPONENTS/WatchNow/";
-import Navigation from "COMPONENTS/Navigation/";
+import Navigation, { NavProvider } from "COMPONENTS/Navigation/";
```

Comment:
did we want to do `{ Provider as NavProvider }` here? Trying to think about the best habits to form, and that seems like a good one...

Reply:
If you feel this is worth doing, I can do it but i'd there would be a fair bit of renaming as it is not how we're doing it elsewhere (for contexts and providers). Seems a bit minor to commit time to it

Reply:
Going to leave this for now.